technologies/BehaviorTree.CPP/README.md

```diff
@@ -2,10 +2,12 @@
 
 This package provides some ready to use Nodes (i.e. "actions") to be used with **pyrobosim**.
 
-We suggest creating your XML trees in the folder `technologies/BehaviorTree.CPP/pyrobosim_btcpp/trees`
+We suggest creating your XML trees in the folder `/delib_ws/src/technologies/BehaviorTree.CPP/pyrobosim_btcpp/trees`
 
 ## How to compile
 
+**Careful**: make sure to run this command in `/delib_ws` folder.
```

Comment:
in *the* delib_ws folder

Reply:
Do we want this command here or use the `delib_build` alias to avoid improper builds?